core/controllers/acl_decorators_test.py

```diff
@@ -5427,7 +5427,7 @@ def post(self):
         },
         'user_supplied_feedback': {
             'report_type': 'suggestion',
-            'category': 'language_suggestion',
+            'category': 'LANGUAGE_SUGGESTION',
```

Comment:
Is this actually correct? I would expect it to be lowercase here. The problem might be that in some place you compare it with `Enum.smth` and not with `Enum.smth.value`.

Reply:
I think at this place, we are comparing it with `Enum.smth` as in the backend tests, it is looking for the value in the Enum(not the value of value). That's why the test was failing

Reply:
Can you fix it so that we compare it with the value of value then?

Reply:
Ditto elsewhere